core/Makefile

```diff
@@ -79,6 +79,15 @@ test_emu_fido2: ## run fido2 device tests
 test_emu_click: ## run click tests
 	cd tests ; ./run_tests_click_emu.sh $(TESTOPTS)
 
+test_emu_ui:  # run ui integration tests
```

Comment:
Single space and double hash, so the correct line is:
`test_emu_ui: ## run ui integration tests`